### ISSUE TITLE
The point of the rules_n_rules folder

### ISSUE THREAD

**Issue by Randium** [werewolves-devs/Werewolf_Bot#41]
For your information, though there currently isn't much going on and we're defining actions directly in ww_head.py, we are still gonna do a lot of useful things with that folder. That folder is going to be used for more complex features. For example, the exorcist's complicated immediate effect of killing someone **IF** they're one of a few specific roles, will be taken care of in there. Same for the fortune teller, when inspecting someone, the function to determine what role they shall be given, will be defined in roles_n_rules\functions.py

TLDR: If a function is getting too complicated for interpretation\check.py, that's when we're defining a function for that in roles_n_rules\functions.py

**Comment by Bentechy66** [comment]
Urgh, this is getting more all over the place by the minute.

**Comment by Randium** [comment]
This one's getting closed, as it should be pretty obvious by now.
